ratelimit/redis.go

```diff
@@ -96,10 +96,11 @@ func (c *clusterLimitRedis) startSpan(ctx context.Context, spanName string) func
 	span.SetTag("max_hits", c.maxHits)
 	span.SetTag("window", c.window.String())
 
-	return func(failed bool) {
+	return func(allowed, failed bool) {
```

Comment:
I think we can just set the tag manually in `AllowContext` and not set it in `oldest` at all

Reply:
We don't have access to span there and then we would need to unpack it there and make startSpan() useless. I don't really care if it's in `oldest` or not, does not hurt to have a true there.

Reply:
> We don't have access to span

we may return it along with function

Reply:
I think I found a better way with an optional args list that is checked.